scripts/checks/architecture/rules/app_page_rules.py

```
@@ -0,0 +1,141 @@
+        page_subsystems = {}
+        for subsystem in subsystems:
+            if subsystem.subsystem_type == "page":
+                subsystem_path = f"~/{subsystem.path.relative_to(Path('src'))}"
+                page_subsystems[subsystem_path] = subsystem
+
+        # Check each page subsystem
+        for page_path, page_subsystem in page_subsystems.items():
+            for file_info in page_subsystem.files:
+                for import_path in file_info.imports:
+                    # Check if importing from another page
+                    for other_page_path, other_page in page_subsystems.items():
```

Ruff (0.14.4)

Loop control variable `other_page` not used within loop body

Rename unused `other_page` to `_other_page`

(B007)